scarlet/interpolation.py

```diff
@@ -206,7 +206,7 @@ def outer(x):
     x = np.abs(dx - window)
     result = np.piecewise(x,
                           [x <= 1, (x > 1) & (x < 2)],
-                          [lambda x:inner(x), lambda x:outer(x)])
+                          [lambda x: inner(x), lambda x: outer(x)])
```

Comment:
it looks like you switched on flake8. Lots of little changes, which we wanted to do in one go in a commit with no functionality change.

Reply:
As I said, this is automated on my ide, but I thought it was pep8 I can check and change that if required.

Reply:
Leave it, but switch off pep8 enforcement in your IDE for now. It makes the changes harder to read.